mobile/src/bundles/common/components/button/button.tsx

```diff
@@ -1,19 +1,122 @@
-import React from 'react';
+import { type IconDefinition } from '@fortawesome/free-solid-svg-icons';
+import {
+    type FontAwesomeIconStyle,
+    type Props,
+} from '@fortawesome/react-native-fontawesome';
+import { FontAwesomeIcon } from '@fortawesome/react-native-fontawesome';
+import React, { useCallback, useState } from 'react';
 import { Pressable } from 'react-native';
+import {
+    type PressableProps,
+    type StyleProp,
+    type ViewStyle,
+} from 'react-native';
 
 import { Text } from '~/bundles/common/components/components';
 
 import { styles } from './styles';
 
 type Properties = {
-    label: string;
-    onPress: () => void;
-};
+    label?: string;
+    isPrimary?: boolean;
+    labelStyles?: StyleProp<ViewStyle>;
```

Comment:
```suggestion
```
You can change text styles depending on the button type

Reply:
it's already got write styles, relaid on props, this labelStyles is rewrite default values if we need.

Reply:
As I see in UI, for now, there'll be no need in rewriting styles for text, but we can add it later if it'll be a need.

Reply:
fix